benchmark/simple-bench.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env sh
+
+rabbit_consumer() {
+  # Injects messages into RabbitMQ, then starts Threat Bus and measures raw consumption times.
+  rm threatbus.log
```

Comment:
```suggestion
  rm -f threatbus.log
```

Do not display non-existant file warnings.